### ISSUE TITLE
requirements_dev.txt

### ISSUE THREAD

**Issue by ghost** [pybee/ouroboros#8]
The requirements_dev.txt that is indicated in the instructions the set up the development environment is not included in the repository. Also the __init__.py containing the version number is missing, and in this case, the `pip install -e .` command can't work.

**Comment by freakboy3742** [comment]
Good catch. The Ouroboros docs are mostly a copy-and-paste of another project, so there are a few bits that don't apply. We need to do a general cleanup of those docs and instructions.
